src/JestExt.ts

```diff
@@ -412,16 +402,27 @@ export class JestExt {
     this.channel.appendLine(noANSI)
   }
 
+  private handleJestEditorSupportEvent(output: string) {
+    if (output.includes('onRunStart')) {
+      this.status.running('Running tests')
+    }
+    if (output.includes('onRunComplete')) {
+      this.status.stopped()
+      this.parsingTestFile = false
+      this.channel.clear()

```

Comment:
hmmm... we probably don't want to clear the output channel at `onRunComplete` since that's when we needed to look at the test results. I think it might be better to do this at the beginning of each run,i.e. `onRunStart`, to clear the previous run's output instead.